cardano-cli/src/Cardano/CLI/Shelley/Run/Query.hs

```diff
@@ -506,7 +506,8 @@ runQueryKesPeriodInfo (AnyConsensusModeParams cModeParams) network nodeOpCertFil
    opCertNodeAndOnDiskCounters o@(OpCertOnDiskCounter odc) (Just n@(OpCertNodeStateCounter nsc))
      | odc < nsc = OpCertOnDiskCounterBehindNodeState o n
      | odc > nsc + 1 = OpCertOnDiskCounterTooFarAheadOfNodeState o n
-     | otherwise = OpCertOnDiskCounterMoreThanOrEqualToNodeState o n
+     | odc == nsc + 1 = OpCertOnDiskCounterAheadOfNodeState o n
```

Comment:
I think this case could be condensed with the following case: `| odc >= nsc + 1 = OpCertOnDiskCounterTooFarAheadOfNodeState o n`  

We can just report how much further ahead it is by rendering `n` and outputting a suitable error message.

Reply:
This would entail reporting success or failure depending on `n`.  `n + 1` is success and `n + 2` or higher is failure.  Are you happy for the decisioning making to be at the point of rendering?

Reply:
`OpCertOnDiskCounterTooFarAheadOfNodeState` would have to be renamed as well because "too far ahead" has bad connotations while `n + 1` is actually ok.

Reply:
Ok, lets leave it how you've done it :+1: